# Commands the trackes of Cozmo
Header  header
float32 vel_left    # velocity of left track
float32 vel_right   # velocity if right track
